tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -456,15 +457,16 @@ def thread_loop(self):
 
             def validate_result():
                 try:
-                    op.validate(self.operation_ctx)
+                    return op.validate(self.operation_ctx)
                 except Exception as e:
-                    self.redpanda.logger.debug(
-                        f"Error validating operation {op_type} - {e}")
+                    self.redpanda.logger.exception(
+                        f"Error validating operation {op_type}")
                     return False
 
             try:
+                self.attempted += 1
                 if self.execute_with_retries(op_type, op):
-                    wait_until(lambda: validate_result,
+                    wait_until(lambda: validate_result(),
```

Comment:
lol, nice catch.
maybe, simply `validate_result` instead of `lambda: validate_result()`?

Reply:
oh gosh. this has happened to me several times in the past